### PR TITLE
Mount all of the repo into the dev-frontend service

### PR DESCRIPTION
**Pull request by mmulich**
In order to get the git information from the frontend code (e.g. run
`git` commands in the `_frontend.codex` directory) the parent project
must be accessible. This is only a problem because the `_frontend.*`
repos are submodules; and thus not actually git repos
themselves (i.e. it's complicated). This solves `fatal: No names
found, cannot describe anything.` when running git commands within the
submodule.

This change is being made, because additions are being made to the
frontend that require git info. The frontend build code discovers its
own version information for display in the server-status page. This
fix is required to enable that feature.

No developer changes required.

### CODE REVIEW

File: docker-compose.codex.yml
Comment (brmscheiner) [inline docker-compose.codex.yml:228]: So I am used to developing locally with a diff here, something like 
```
/path/to/my/repo:/code
```
Do I need to move my repo into a `code/_frontend` subfolder now? Or what would be the best way to do it

Comment (mmulich) [reply docker-compose.codex.yml:228]: Two ways to solve this.
1. change the `working_dir` setting in this service.
2. adjust the entrypoint script to be somewhat aware of the situation.

I've gone with option two because it means one less change the developer would need to fiddle with. So I've adjusted the script to check for the existence of the submodule directory. If the directory exists, it'll change to to it, otherwise it'll assume the code is in the current working directory.

Comment (brmscheiner) [reply docker-compose.codex.yml:228]: Oh that's great!

Comment (brmscheiner) [reply docker-compose.codex.yml:228]: I'll approve but I'd wait for another approval from a backend person

**Review by brmscheiner — APPROVED** [review]

**Pull request merged**
